### PR TITLE
Center table at BSX stats and handle longer item names

### LINKED ISSUE #3378: Center table at BSX stats and handle longer item names
Would be nice to limit length of item name to 50 characters

Seems we are not reading all items' names?
edit, due to Estuary issues in past, we have not indexed all the stuff there. 
✅ seems the new one works now well.

https://deploy-preview-3373--koda-nuxt.netlify.app/bsx/stats/

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3378
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/bsx/Offer/OfferTable.vue [hunk 1]
```diff
@@ -54,8 +54,9 @@
       v-slot="props"
       sortable>
       <nuxt-link :to="`gallery/${props.row.nft.id}`">
-        <p v-if="props.row.nft.name">{{ props.row.nft.name }}</p>
-        <p v-else>{{ props.row.nft.id }}</p>
+        <p class="limit-width-text">
```
Comment (prachi00) [inline components/bsx/Offer/OfferTable.vue:57]: can we add `title` here as if the character is more than 50, we wont be able to see the full name if we hover over

**Review by prachi00 — APPROVED** [review]

**Comment by prachi00** [comment]
lgtm

**Comment by prachi00** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 49.39 USD/KSM ~ 1.012 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x760921a5af1840d3015da66f6d9f2fbef9c9d5c5bd4b32290686174d5055f6cc](https://kusama.subscan.io/extrinsic/0x760921a5af1840d3015da66f6d9f2fbef9c9d5c5bd4b32290686174d5055f6cc)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**